subsys/bluetooth/controller/ll_sw/ctrl_internal.h

```diff
@@ -90,18 +90,25 @@ struct connection {
 	u16_t apto_expire;
 #endif /* CONFIG_BLUETOOTH_CONTROLLER_LE_PING */
 
-	struct {
-		u8_t  sca:3;
-		u8_t  latency_cancel:1;
-		u32_t window_widening_periodic_us;
-		u32_t window_widening_max_us;
-		u32_t window_widening_prepare_us;
-		u32_t window_widening_event_us;
-		u32_t window_size_prepare_us;
-		u32_t window_size_event_us;
-		u32_t force;
-		u32_t ticks_to_offset;
-	} slave;
+	union {
```

Comment:
It had to be dont in separate commit and it is clear the way it is commit now (with the use of anonymous union here).